core/ecschema-editing/src/Editing/SchemaItems.ts

```diff
@@ -27,6 +28,32 @@ export abstract class SchemaItems {
     this.schemaEditor = schemaEditor;
   }
 
+  /**
+   * Sets the SchemaItem description.
+   * @param schemaItemKey The SchemaItemKey of the SchemaItem
+   * @param description The new description to set.
+   */
+  public async setDescription(schemaItemKey: SchemaItemKey, description: string) {
+    const item = await this.getSchemaItem<MutableSchemaItem>(schemaItemKey)
+      .catch((e: any) => {
+        throw new SchemaEditingError(ECEditingStatus.SetDescription, new SchemaItemId(this.schemaItemType, schemaItemKey), e);
+      });
+    item.setDescription(description);
+  }
+
+  /**
+   * Sets the SchemaItem display label.
+   * @param schemaItemKey The SchemaItemKey of the SchemaItem
+   * @param label The new label to set.
+   */
+  public async setDisplayLabel(schemaItemKey: SchemaItemKey, label: string) {
+    const item = await this.getSchemaItem<MutableSchemaItem>(schemaItemKey)
+      .catch((e: any) => {
+        throw new SchemaEditingError(ECEditingStatus.SetLabel, new SchemaItemId(this.schemaItemType, schemaItemKey), e);
+      });
+    item.setDisplayLabel(label);
+  }
+
```

Comment:
If I understand that right, we provide the editing methods here in an "unspecified" item-space, but don't call this ourself anywhere yet. I guess the Merger will call it though once available which can save us to cast to the mutables at some places.

However, if we don't call it here (in this PR scope), we might want to have an own PR for that in which we can also change the merger code.

Reply:
Carole requested these methods (she actually requested the label, but I added description too).  Up to this point, unless we have to due to failing tests in the merger, I've kept editing API code changes separate from any merger changes.